tests/unit/AlterationSummary.spec.ts

```diff
@@ -8,6 +8,7 @@ import AlterationSummary from '@/components/Alteration/AlterationSummary.vue'
 import { ConfirmDialog as ConfirmDialogShared } from '@bcrs-shared-components/confirm-dialog/'
 import EffectiveDateTime from '@/components/common/EffectiveDateTime.vue'
 import NameTranslation from '@/components/common/YourCompany/NameTranslations/NameTranslation.vue'
+import { FeesIF } from '@/interfaces'
```

Comment:
No. I think the unit tests should not know about interfaces. They should be hard-coded. This will make them more fragile and pre-empt the possibility of 2 wrongs canceling each other out.

Reply:
Ok, I added it in there, because the any object was being assigned an object (FeesIF) instead of an array FeesIF[]

I'll revert those changes